docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -21,14 +21,13 @@ The `SignIn` object's properties can be split into logical groups, with each gro
-| `supportedFirstFactors` | `SignInFactor[]` | Array of the first factors that are supported in the current sign-in. Each factor contains information about the verification strategy that can be used.<br/>For example: <ul><li>`email_code` for email addresses</li><li>`phone_code` for phone numbers</li></ul> As well as the identifier that the factor refers to. |
-| `supportedSecondFactors` | `SignInFactor[]` | Array of the second factors that are supported in the current sign-in. Each factor contains information about the verification strategy that can be used.<br/>For example: <ul><li>`email_code` for email addresses</li><li>`phone_code` for phone numbers</li></ul> As well as the identifier that the factor refers to. Please note that this property is populated only when the first factor is verified. |
-| `firstFactorVerification` | `Verification` | The state of the verification process for the selected first factor. Please note that this property contains an empty verification object initially, since there is no first factor selected. You need to call the [`prepareFirstFactor`](/docs/references/javascript/sign-in/first-factor#prepare-first-factor) method in order to start the verification process. |
-| `secondFactorVerification` | `Verification` | The state of the verification process for the selected second factor. Similar to `firstFactorVerification`, this property contains an empty verification object initially, since there is no second factor selected. For the `phone_code` strategy, you need to call the [`prepareSecondFactor`](/docs/references/javascript/sign-in/second-factor#prepare-second-factor) method in order to start the verification process. For the `totp` strategy, you can directly attempt. |
+| `supportedFirstFactors` | `SignInFirstFactor[]` | Array of the first factors that are supported in the current sign-in. Each factor contains information about the verification strategy that can be used. See the [`SignInFirstFactor`](/docs/references/javascript/types/sign-in-first-factor) type reference for more information. |
+| `supportedSecondFactors` | `SignInSecondFactor[]` | Array of the second factors that are supported in the current sign-in. Each factor contains information about the verification strategy that can be used. This property is populated only when the first factor is verified. See the [`SignInSecondFactor`](/docs/references/javascript/types/sign-in-second-factor) type reference for more information. |
+| `firstFactorVerification` | `Verification` | The state of the verification process for the selected first factor. Initially, this property contains an empty verification object, since there is no first factor selected. You need to call the [`prepareFirstFactor`](/docs/references/javascript/sign-in/first-factor#prepare-first-factor) method in order to start the verification process. |
```

Comment:
Random comment not related to this line: We need to add the two new pages to `manifest.json`, no?

Reply:
WOOPS! yes :P